gtests/net/mptcp/common/multi-ep.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+#
+# Assign additional (advertised) addresses to the local device,
+# add a local/remote subflow for each address
+
+usage() { echo "$0 [-e <endpoints>] [-m <signal|subflow>] [-b]" 1>&2; exit 1; }
+
+create_endpoints() {
+    local host= flags= ep=1 max=$1 hn=$2 nn=$3 pl=$4 sig=${5:-0} bck=${6:-0}
+
+
+    [ $sig -eq 1 ] && flags=${flags:+$flags }signal || flags=${flags:+$flags }subflow
+    [ $bck -eq 1 ] && flags=${flags:+$flags }backup
+
+    ip mptcp endpoint flush
+    while [ ${ep} -le ${max} ]; do
+        if [ "$OPT_IP_VERSION" = "ipv6" ]; then
+            host=$(printf "%s:%x" "${nn}" "$(($(printf '%d' 0x${hn})+ep))")
+        else
+            host=${nn}$((hn+ep))
+        fi
+        ip addr add $host/$pl dev $OPT_LOCAL_DEV
+        ip mptcp endpoint add $host $flags
+	ep=$((ep+1))
```

Comment:
(detail: it looks like you have a tab here and not spaces like in the rest of the document.)

Reply:
correct: fixed